### PR TITLE
Add objects for NFTokenPages

### LINKED ISSUE #430: Add NFTokenPage Object
Previously, the `NFTokenPage` ledger object was inaccessible via xrpl4j because it required a `ledger_entry` RPC which we don't support (yet). Therefore, we never added a `LedgerObject` for `NFTokenPage`. However, once https://github.com/XRPLF/rippled/pull/4352 is running on ledger, `NFTokenPage`s will show up in responses to `account_objects`.

### PR DESCRIPTION
**Pull request by nkramer44**
Fixes #430 and #431

### CODE REVIEW

File: xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/flags/NfTokenFlags.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+import org.xrpl.xrpl4j.model.transactions.AccountSet;
+
+/**
+ * A set of static {@link Flags} which can be set on {@link AccountSet} transactions.
```
Comment (sappenin) [inline xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/flags/NfTokenFlags.java:26]: This comment is incorrect.

File: xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountObjectsRequestParams.java [hunk 2]
```diff
@@ -61,6 +61,7 @@ static ImmutableAccountObjectsRequestParams.Builder builder() {
   static AccountObjectsRequestParams of(Address classicAddress) {
     return builder()
       .account(classicAddress)
+      .ledgerSpecifier(LedgerSpecifier.CURRENT)
```
Comment (sappenin) [inline xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountObjectsRequestParams.java:64]: Why is `CURRENT` the chosen default here?

**Review by sappenin — CHANGES_REQUESTED** [review]

File: xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountObjectsRequestParams.java (hunk 2 above)
Comment (nkramer44) [reply xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountObjectsRequestParams.java:64]: This was actually a bug that I found that we didn't have test coverage for. Before this change, `AccountObjectsRequestParams.of` would throw an `IllegalStateException` always. Many of the other request params  classes (ie `AccountInfoRequestParams`) have a similar method, and they all default to `LedgerSpecifier.CURRENT`, so this is consistent with those.

Comment (sappenin) [reply xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountObjectsRequestParams.java:64]: That makes sense -- however, shouldn't we choose `VALIDATED` in the default case?

Comment (nkramer44) [reply xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountObjectsRequestParams.java:64]: After speaking with @sappenin, we decided to potentially change the default from `CURRENT` to `VALIDATED` in v4 (see #426), as changing this default would require changing other defaults, which would be a breaking change.

**Pull request merged**